Sprint-2/interpret/time-format.js

```diff
@@ -20,12 +18,27 @@ function formatTimeDisplay(seconds) {
 
 // a) When formatTimeDisplay is called how many times will pad be called?
 
+// a) 🔄 pad is called 3 times:
+
+// Once for hours
+// Once for minutes
+// Once for seconds
+
+// This is because the time format is "HH:MM:SS" and each number needs padding!
+
 // Call formatTimeDisplay with an input of 61, now answer the following:
 
-// b) What is the value assigned to num when pad is called for the first time?
+// b) What is the value assigned to num when pad is called for the first time? 0
 
-// c) What is the return value of pad is called for the first time?
+// c) What is the return value of pad when pad is called for the first time?  "00"
 
 // d) What is the value assigned to num when pad is called for the last time in this program?  Explain your answer
 
+// d) The value assigned to num when pad is called for the last time in this program is 1.
+// This is because pad is called with the value of remainingSeconds, which is 1, or 61 % 60.
+// 61 seconds is 1 minute and 1 second in total.
+
 // e) What is the return value assigned to num when pad is called for the last time in this program?  Explain your answer
```

Comment:
I don't love the wording of this question. How would you feel about: "What does the pad function return the last time it is called in this program?"?

Reply:
Happy to change the questions but probably not in the solutions! That would be annoying for the learner. 

I think overall this module needs slimming down a LOT - there's too much work here and people won't have time to think about what they are doing or really explore and enjoy the work. I ploughed through this in between meetings and emails and a broken down boiler and I got super bored and inattentive.

Reply:
Yeah - personally I'd be pretty happy to remove most of the "playing computer" exercises - I think they're quite long-winded, fiddly to assess, and it's a better exercise to do in person with a person interacting with you to say "you skipped a line, slow down, I know it's frustrating"...

(An agree about changing in the questions not the solutions!)